lighty-core/lighty-controller/src/main/java/io/lighty/core/controller/api/AbstractLightyModule.java

```diff
@@ -184,5 +185,17 @@ public synchronized ListenableFuture<Boolean> shutdown() {
 
         return shutdownFuture;
     }
-}
 
+    @SuppressWarnings("IllegalCatch")
+    public final boolean shutdown(final long duration, final TimeUnit unit) {
```

Comment:
This method can be used only if it will be implemented in LightyModule. Otherwise, example application will have to retype objects to AbstractLightyModule e.g:
((AbstractLightyModule) restconf).shutdown()

Reply:
Done.